### DISCUSSION TITLE
Has anyone used socket.io on a Begin app?

### DISCUSSION THREAD

**Discussion by spencerflagg** [beginner-corp/begin-community#72] [Q&A] [answered]
The Socket.IO server normally exposes a client bundle at `/socket.io/socket.io.js`.
It gets exposed when I spin up the project locally, but not up on Begin staging.

New to socket.io; I'm sure I'm missing something obvious.

Thanks in advance!

**Comment by brianleroux** [comment, marked as answer]
Hey @spencerflagg we have not enabled web sockets in our free tier however this is something you can do with our upcoming Pro+ tier of service. If you'd like to preview working with web sockets you can check out the open source stuff which does work locally or deployed to your own AWS account. https://arc.codes/docs/en/reference/app.arc/ws (yes it is that simple…no servers or legacy libs like socket.io needed!!)

**Reply by spencerflagg** [reply]
Ah - thanks. I probably should have done my research before making the attempt.

**Reply by brianleroux** [reply]
All good; let us know if you need/want help getting started with the open source project and your own AWS. Otherwise stay tuned! Web socket support is coming to Begin proper.
